run.sh

```diff
@@ -32,6 +32,8 @@ java $JAVA_OPTS \
  -Drestolino.classes=$RESTOLINO_CLASSES \
  -Drestolino.packageprefix=$PACKAGE_PREFIX \
  -DSTART_EMBEDDED_SERVER=N \
+  -javaagent:zebedee-cms/target/dependency/aws-opentelemetry-agent-1.30.0.jar \
```

Comment:
Should this be aws-opentelemetry-agent-1.31.0.jar for consistency with the babbage version?